manubot/process/header-includes-template.html

```diff
@@ -0,0 +1,75 @@
+<!--
+Manubot generated metadata rendered from header-includes-template.html.
+Suggest improvements at https://github.com/manubot/manubot/blob/master/manubot/process/header-includes-template.html
+-->
+<meta name="dc.format" content="text/html" />
+{% if pandoc.title is defined -%}
+<meta name="dc.title" content="{{ pandoc.title|escape }}" />
+<meta name="citation_title" content="{{ pandoc.title|escape }}" />
+<meta property="og:title" content="{{ pandoc.title|escape }}" />
+<meta property="twitter:title" content="{{ pandoc.title|escape }}" />
+{% endif -%}
+{% if pandoc['date-meta'] is defined -%}
+<meta name="dc.date" content="{{ pandoc['date-meta'] }}" />
+<meta name="citation_publication_date" content="{{ pandoc['date-meta'] }}" />
+{% endif -%}
+{% if pandoc.lang is defined -%}
+<meta name="dc.language" content="{{ pandoc.lang }}" />
+<meta name="citation_language" content="{{ pandoc.lang }}" />
+{% endif -%}
+<meta name="dc.relation.ispartof" content="Manubot Preprint" />
```

Comment:
In https://github.com/manubot/rootstock/issues/176#issuecomment-461085068 you pointed out that not all Manubot documents are "preprints".  I now agree with that.  Should we be more general here and below?

Reply:
In https://github.com/manubot/manubot/pull/138/commits/86ef608121c88a95c6d3c124a20fb8c17a6e9d12, I revert to Manubot.

Reply:
Manubot is also what I would have recommended.  It is such a general writing platform that even "manuscripts" doesn't exactly include documents like grants.